packages/module/src/SourcesCard/SourcesCard.tsx

```diff
@@ -84,167 +73,15 @@ export interface SourcesCardProps extends CardProps {
-                    data-action="previous"
-                    onClick={(event) => {
-                      const newPage = page >= 1 ? page - 1 : 1;
-                      onPreviousClick && onPreviousClick(event, newPage);
-                      handleNewPage(event, newPage);
-                    }}
-                    aria-label={toPreviousPageAriaLabel}
-                  >
-                    <Icon iconSize="lg">
-                      {/* these are inline because the viewBox that works in a round icon is different than the PatternFly default */}
-                      <svg
-                        className="pf-v6-svg"
-                        viewBox="0 0 280 500"
-                        fill="currentColor"
-                        aria-hidden="true"
-                        role="img"
-                        width="1em"
-                        height="1em"
-                      >
-                        <path d="M31.7 239l136-136c9.4-9.4 24.6-9.4 33.9 0l22.6 22.6c9.4 9.4 9.4 24.6 0 33.9L127.9 256l96.4 96.4c9.4 9.4 9.4 24.6 0 33.9L201.7 409c-9.4 9.4-24.6 9.4-33.9 0l-136-136c-9.5-9.4-9.5-24.6-.1-34z"></path>
-                      </svg>
-                    </Icon>
-                  </Button>
-                  <span aria-hidden="true">
-                    {page}/{sources.length}
-                  </span>
-                  <Button
-                    variant={ButtonVariant.plain}
-                    isDisabled={isDisabled || page === sources.length}
-                    aria-label={toNextPageAriaLabel}
-                    data-action="next"
-                    onClick={(event) => {
-                      const newPage = page + 1 <= sources.length ? page + 1 : sources.length;
-                      onNextClick && onNextClick(event, newPage);
-                      handleNewPage(event, newPage);
-                    }}
-                  >
-                    <Icon isInline iconSize="lg">
-                      {/* these are inline because the viewBox that works in a round icon is different than the PatternFly default */}
-                      <svg
-                        className="pf-v6-svg"
-                        viewBox="0 0 180 500"
-                        fill="currentColor"
-                        aria-hidden="true"
-                        role="img"
-                        width="1em"
-                        height="1em"
-                      >
-                        <path d="M224.3 273l-136 136c-9.4 9.4-24.6 9.4-33.9 0l-22.6-22.6c-9.4-9.4-9.4-24.6 0-33.9l96.4-96.4-96.4-96.4c-9.4-9.4-9.4-24.6 0-33.9L54.3 103c9.4-9.4 24.6-9.4 33.9 0l136 136c9.5 9.4 9.5 24.6.1 34z"></path>
-                      </svg>
-                    </Icon>
-                  </Button>
-                </nav>
-              </div>
-            </CardFooter>
-          )
-        )}
-      </Card>
-    </div>
-  );
-};
+}: SourcesCardProps) => (
+  <div className="pf-chatbot__source">
```

Comment:
Do we want to nest SourcesCardBase which also has this div wrapper inside of here, or should only 1 have this div wrapper?